bin/cml/runner/launch.js

```diff
@@ -582,6 +584,11 @@ exports.options = kebabcaseKeys({
     description: 'Specifies the subnet to use within AWS',
     alias: 'cloud-aws-subnet-id'
   },
+  cloudImage: {
+    type: 'string',
+    description: 'Custom machine/container image',
+    hidden: true
```

Comment:
qq @0x2b3bfa0  - should this be documented in https://cml.dev/doc/ref/runner?

Reply:
Because https://github.com/iterative/cml/pull/1302#issuecomment-1369117730

Reply:
Scared to make that part of the public API and having to maintain a footgun.

Reply:
I can understand:

1. implementing X.
2. not implementing X.
3. implementing X but hiding it as a beta feature.

Sounds like `--cloud-image` falls into none of the above categories? I'm confused.

Reply:
Sounds like 3 to me: implemented but not officially supported for now,